app/_config/mimevalidator.yml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
This file was copied from CWP. If you upgrade from recipe-core 4.5 to 4.6, our recipe-plugin will copy this file directly into your project. This will enable mime validation for everyone running recipe-core even if they created their project from an older release. You'll need to commit the file into your own repo.

If you upgrade from a CWP project, you'll already have that file. recipe-plugin won't try overriding it, so that shouldn't be a problem.